src/plugins/console/public/lib/autocomplete/autocomplete.ts

```diff
@@ -998,12 +998,12 @@ export default function ({
       ) {
         // will simulate autocomplete on 'GET /a/b/' with a filter by index
         return {
-          tokenPath: context.urlTokenPath.slice(0, -1),
-          predicate: (term) => term.meta === 'index',
+          tokenPath: context?.urlTokenPath?.slice(0, -1),
```

Comment:
nit: We might not need the first `?` since `context` is not an optional parameter.

```suggestion
          tokenPath: context.urlTokenPath?.slice(0, -1),
```